auth_otp/services/otp_service.py

```
@@ -0,0 +1,341 @@
+                
+                logger.info(
+                    f"OTP verified successfully: {phone_number}, "
+                    f"purpose: {purpose}"
+                )
+                
+                return True, {
+                    'otp_request': otp_request,
+                    'message': 'کد تأیید با موفقیت تأیید شد'
+                }
+                
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)